.travis.yml

```diff
@@ -9,3 +9,4 @@ python:
 # install: "sudo apt-get install"
 # command to run tests, e.g. python setup.py test
 script: python cpplint_unittest.py
+script: python nitpick_unittest.py
```

Comment:
I got the emails, clicked one of them and I do see it - yey.
I'm adding a comment from the diff view